cmake/Modules/FindTPLCUDA.cmake

```diff
@@ -1,17 +1,37 @@
+IF (NOT CUDAToolkit_ROOT)
+  IF (NOT CUDA_ROOT)
+    SET(CUDA_ROOT $ENV{CUDA_ROOT})
+  ENDIF()
+  IF(CUDA_ROOT)
+    SET(CUDAToolkit_ROOT ${CUDA_ROOT_ENV})
```

Comment:
Type-o by me. `CUDA_ROOT_ENV` should be `CUDA_ROOT`. Transferring comment from PR #3028.

Reply:
Done